set.mm

```diff
@@ -425322,96 +425387,97 @@ with the first (and therefore last) vertex to the pair of the shorter
+        UIRUJZEUKUAULZEQZUBULZEQZSZUUIUUKSZUMZUBUUFVCUAUUFVCUUFUUHEUNABCDEFGHIJ
+        KLMNOUOUUEUUOUAUBUUFUUFUUEUUIUUFPZUUKUUFPZTZTZUUMUUIUPUUGUQZURRZIUSUHRZ
+        UUIQZUQZUUKUUTURRZUVBUUKQZUQZSZUUNUUSUUJUVDUULUVGUUPUUJUVDSUUEUUQABCDUU
+        IEFGHIJKLMNOUTVAUUQUULUVGSUUEUUPABCDUUKEFGHIJKLMNOUTVBVDUVHUVAUVESZUVCU
+        VFSZTZUUSUUNUVAUVCUVEUVFUUIUUTURVEUVBUUIVFVMUUEUURUVKUUNUMZUUEUUPUUIJVG
+        ZPZUUIVHQZISZTZUPUUIQZKSZUUGUUIQZUVRSZTZTZUUQUUKUVMPZUUKVHQZISZTZUPUUKQ
+        ZKSZUUGUUKQZUWHSZTZTZUVLUUBUUDUUPUWCUMUUAUUBUUDTZUUPUUIIHVIRZPZUVSUWAUF
+        ZUWCUUDUUBIUGUEQPZUUPUWQVJIVKZABDUUIFHIJKNVLVNUWQUVQUWBUWPUVSUVQUWAHIJU
+        UILVOVPUWPUVSUWAVQVRVSVTUUBUUDUUQUWMUMUUAUWNUUQUUKUWOPZUWIUWKUFZUWMUUDU
+        UBUWRUUQUXAVJUWSABDUUKFHIJKNVLVNUXAUWGUWLUWTUWIUWGUWKHIJUUKLVOVPUWTUWIU
+        WKVQVRVSVTUUEUWCUWMTZUUNUVKUUEUXBTZUUNUVOUWESZUUIUPUVOUGUHRZUQZURRZUUKU
+        XFURRZSZUXEUUIQZUXEUUKQZSZUUIWDQZUUKWDQZSZUFZTZUXPUVKUXCUVNUWDUSUVOWAWB
+        ZUUNUXQVJUWCUVNUUEUWMUVNUVPUWBWCVAUXBUWDUUEUWCUWDUWFUWLWEWFUXBUUEUXRUVP
+        UUEUXRUMUVNUWBUWMUUEUVPUXRUUDUUAUVPUXRUMUUBUVPUUDUXRUVPUUDUVOUUCPZUXRUU
+        DUXSVJIUVOIUVOUUCWGWHUXSUDWIPZUVOWIPZUDUVOWJWBZUFUXRUDUVOWKUYAUYBUXRUXT
+        UYAUYBUXRUYAUSWLPZUDWLPZUVOWLPZUFZUSUDWAWBZUYBUXRUMUYAUYCUYDUYEUYAWMUYD
+        UYAWNWOUVOWRWPWQUYFUYGUYBUXRUSUDUVOWSWTXAXBVTXCVSXHXDXHXEXFUUKJUUIXGXIU
+        XCUXDUXPUXBUXDUUEUWCUWMUXDUVPUWMUXDUMUVNUWBUWMUVPUXDUWFUVPUXDUMUWDUWLUV
+        PUWFUXDUVOUWEIXJXKXLXHXLXBWFXMUXCUXPUXLUXIUXOTZTZUYHUVKUXPUYIVJUXCUXIUX
+        LUXOXRWOUXCUXLUYHUXBUXLUUEUXBUXJKUXKUWCUXJKSZUWMUVQUWBUYJUVPUWBUYJUMUVN
+        UWBUVPUYJUVSUWAUVPUYJUMZUVSUWAUVTKSZUYKUVRKUVTXNUVPUYLUYJUVPUVTUXJKUVPU
+        UGUXEUUIUUGUXESIUVOIUVOUGUHXOWHXPXQXSVSXBXHWFXBXTUVPUWMUXKKSUVNUWBUVPUW
+        MUXKUWJKUVPUXEUUGUUKUVOIUGUHXOZXPUWLUWJKSZUWGUWKUWIUYNUWIUYNVJUWHUWJUWH
+        UWJKYAWHYHWFYBYCYDWFXMUXCUXIUVIUXOUVJUXBUXIUVIVJZUUEUVPUYOUVNUWBUWMUVPU
+        XGUVAUXHUVEUVPUXFUUTUUIURUVPUXEUUGUPUYMYEZYFUVPUXFUUTUUKURUYPYFVDXEWFUX
+        BUXOUVJVJUUEUWCUWMUXMUVCUXNUVFUVQUXMUVCSUWBUVNUVPUXMUVOUSUHRZUUIQUVCUUI
+        UVMYGUVPUYQUVBUUIUVOIUSUHXOXPYBXTUWGUXNUVFSZUWLUWGUYRUXNUWEUSUHRZUUKQZS
+        ZUWDVUAUWFUUKUVMYGXTUWFUYRVUAVJUWDUWFUVFUYTUXNUWFUVBUYSUUKUVBUYSSIUWEIU
+        WEUSUHXOWHXPYIWFYJXTYKWFYLYQYQYMYRXBYNYOYPUAUBUUFUUHEYSYT $.
 
       $d C a b p x $.  $d F a b p x $.  $d G a b p w x $.  $d N a b p x $.
-      $d T a b p x $.  $d V a b p x $.  $d X a b p x $.  $d a i $.
+      $d T a b p x $.  $d V a b p x $.  $d X a b p x $.  $d a i $.  $d b u $.
       $( ` T ` is an onto function.  (Contributed by Alexander van der Vekens,
          20-Sep-2018.)  (Revised by AV, 29-May-2021.) $)
       numclwlk1lem2fo $p |- ( ( G e. USGraph /\ X e. V
                                    /\ N e. ( ZZ>= ` 3 ) )
                  -> T : ( X C N )
                         -onto-> ( ( X F ( N - 2 ) ) X. ( G NeighbVtx X ) ) ) $=
```

Comment:
you can probably write the statement on three lines instead of four.

Reply:
Good suggestion, I'll change that.

Reply:
Done

Reply:
maybe even two lines !
```
      numclwlk1lem2fo $p |- ( ( G e. USGraph /\ X e. V /\ N e. ( ZZ>= ` 3 ) )
       -> T : ( X C N ) -onto-> ( ( X F ( N - 2 ) ) X. ( G NeighbVtx X ) ) ) $=
```

Reply:
With two lines only the statement is too compressed and is not clearly apparent anymore (squeezed between the comment and the proof...).